src/json_parser.cc

```diff
@@ -77,4 +77,26 @@ std::optional<std::string> JSONParser::GetTopLevelField(
   return utf8_value.ToString();
 }
 
+std::optional<bool> JSONParser::GetTopLevelBoolField(const std::string& field) {
+  Isolate* isolate = isolate_.get();
+  Local<Context> context = context_.Get(isolate);
+  Local<Object> content_object = content_.Get(isolate);
+  Local<Value> value;
+  // It's not a real script, so don't print the source line.
+  errors::PrinterTryCatch bootstrapCatch(
+      isolate, errors::PrinterTryCatch::kDontPrintSourceLine);
+  if (!content_object
+           ->Has(context, OneByteString(isolate, field.c_str(), field.length()))
+           .FromMaybe(false)) {
+    return false;
+  }
+  if (!content_object
+           ->Get(context, OneByteString(isolate, field.c_str(), field.length()))
```

Comment:
I don’t think you can assume that `field` is meant to be ISO-8859-1/ASCII in a generic utility function.

Reply:
Yeah, I guess … can you leave a `TODO` comment?

Reply:
@addaleax done, PTAL!